pkg/cmd/pkg/package_repo_add.go

```diff
@@ -13,7 +15,16 @@ func newCmdPackageRepoAdd(ctx api.Context) *cobra.Command {
 		Short: "Add a package repository to DC/OS",
 		Args:  cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			repoName := args[0]
+			repoURL := args[1]
+
+			c, err := cosmos.NewClient(ctx, pluginutil.HTTPClient(""))
+			if err != nil {
+				return err
+			}
+
+			_, err = c.PackageAddRepo(repoName, repoURL, index)
```

Comment:
We need to make the distinction between passing `--index=0` and not passing `--index` at all. See https://docs.d2iq.com/mesosphere/dcos/1.13/deploying-services/package-api/#/default/POST_package_repository_add : 

> To add a package repository to the beginning of the list set the index to zero (0). To add a package repository to the end of the list do not specify an index.

But here we'd always be passing 0. The `PackageAddRepo()` method needs to take a pointer for the index. To check if the flag is passed with 0 or not passed at all we can use https://godoc.org/github.com/spf13/pflag#FlagSet.Changed